Update developer environment section in README (#195)
nginx/agent: README.md

sudo apt install golang-go
```

Install NGINX Agent tools and dependencies:

> Before starting development on the NGINX Agent, it is important to download and install the necessary tool and dependencies required by the NGINX Agent. This can be done by running the following `make` command:

Reviewer: ```suggestion
Before starting development on the NGINX Agent, it is important to download and install the necessary tool and dependencies required by the NGINX Agent. You can do this by running the following `make` command:
```

Avoid passive voice when possible.
